src/retail_calendar.ts

```diff
@@ -149,25 +149,29 @@ export const RetailCalendarFactory: RetailCalendarConstructor = class Calendar
     let weeksInQuarter = 0
```

Comment:
This value is no longer read. Should be safe to remove.

Reply:
This is fixed!